bjklemme-intel/fl-tutorial
# [Bug: 766] Fix OpenFL MICCAI tutorials

Fixes following 2 notebook in the tutorials -
1) 2D MedMNIST - added ipywidgets to make tqm work on jupyter notebook 
2) OpenFL GaNDLF tutorial - added note that OpenFL needs python 3.8 to work without any errors. Google collab runs the notebook with python 3.9.16 by default, this needs to be changed to 3.8 to run without any errors 

Fixes [#766](https://github.com/securefederatedai/openfl/issues/766)

**Testing Steps**

- Download code
- got to tutorial_notebooks
- run `juptyer notebook`
- open notebooks` 2D MedMNIST` & `OpenFL GaNDLF tutorial`
- note on using python 3.8 should be seen on the top
- both notebooks should run without errors

Co-authored-by: Emmanuel Jillela <emmanuel.jillela@intel.com>
Signed-off-by: Neethu Elizabeth Simon <neethu.elizabeth.simon@intel.com>
